pkg/storage/endpoint/resource_group.go

```diff
@@ -22,7 +22,7 @@ import (
 
 // ResourceGroupStorage defines the storage operations on the rule.
 type ResourceGroupStorage interface {
-	LoadResourceGroups(f func(k, v string)) error
+	LoadResourceGroups(prefix string, f func(k, v string)) error
```

Comment:
It seems that this prefix remains the same all the time, can we make it internal rather than passing it all around?

Reply:
In addition to setting, we may load other state information in the future.  But maybe we can change the name. Do you have any suggestions?

Reply:
We could name it `LoadResourceGroupsSetting` to prevent exposing the prefix. If we want to load other info in the future, we could also create a new function like `LoadResourceGroupsXXX`.